test/unit/modules.spec.js

```diff
@@ -124,6 +125,49 @@ describe('Modules', () => {
       store.commit('a/foo')
       expect(mutationSpy).toHaveBeenCalled()
     })
+
+    it('should keep getters when component gets destroyed', async () => {
+      const store = new Vuex.Store()
+
+      const moduleA = {
+        namespaced: true,
+        state: () => ({ value: 1 }),
+        getters: {
+          getState: (state) => state.value
```

Comment:
You could also pass a spy and count the number of times it's called

Reply:
Great idea! Added! 👍

Reply:
Done!